langchain4j/src/main/java/dev/langchain4j/service/DefaultAiServices.java

```diff
@@ -124,7 +125,17 @@ public Object invoke(Object proxy, Method method, Object[] args) throws Exceptio
 
                         validateParameters(method);
 
-                        Object memoryId = findMemoryId(method, args).orElse(DEFAULT);
+                        Object memoryId = findMemoryId(method, args)
+                                .map(mid -> {
+                                    if (null == context.chatMemoryProvider) {
+                                        throw illegalConfiguration(
+                                                "The ChatMemoryProvider configuration with @MemoryId in method '%s' must not be null",
```

Comment:
```suggestion
                                                "In order to use @MemoryId, please configure the ChatMemoryProvider on the '%s'.",
```
%s should refer to the fully qualified class name of the AI Serivce